src/garage/envs/wrappers/multi_env_wrapper.py

```diff
@@ -0,0 +1,147 @@
+"""A wrapper env that handles multiple tasks from different envs.
+
+Useful while training multi-task reinforcement learning algorithms.
+It provides observations augmented with one-hot representation of tasks.
+"""
+
+import random
+
+import akro
+import gym
+import numpy as np
+
+
+class SampleStrategy:  # pylint: disable=too-few-public-methods
+    """Base sample stretegy class.
+
+    Args:
+        num_tasks (int): Total number of tasks.
+
+    """
+
+    def __init__(self, num_tasks):
+        self._num_tasks = num_tasks
+
+    def sample_task(self):
+        """Sample new task."""
+        pass
+
+
+class RoundRobinStrategy(SampleStrategy):  # noqa: E501 # pylint: disable=too-few-public-methods
```

Comment:
i am open to disabling this rule, but i think it did a good thing by pointing out how little logic this class holds. The intent of this rule is to prevent the proliferation of tons of single-use classes (Python isn't Java, so usually there are far fewer classes).

What if you changed SamplingStrategy from a class to a function?

```python
def round_robin_sampling(num_tasks, last_task):
    if last_task is None:
        return 0

    return (last_task + 1) % num_tasks


def uniform_random_strategy(num_tasks, _):
    return random.randint(0, num_tasks - 1)
```

if sampling gets complicated-enough to require a class, we can revisit that.